src/metadata.rs

```diff
@@ -434,6 +465,15 @@ impl AuthorizationMetadata {
         self.0.authorization_endpoint()
     }
 
+    pub fn pushed_authorization_endpoint(&self) -> AuthUrl {
+        // TODO find better way to avoid unwrap
+        self.0
+            .additional_metadata()
+            .clone()
+            .pushed_authorization_request_endpoint
+            .unwrap()
+    }
```

Comment:
There's no simple way to avoid the unwrap, so I would suggest returning `Option<AuthUrl>`